google/cloud/aiplatform/training_jobs.py

```diff
@@ -1653,14 +1829,35 @@ def run(
             accelerator_count (int):
                 The number of accelerators to attach to a worker replica.
             training_fraction_split (float):
-                The fraction of the input data that is to be
-                used to train the Model. This is ignored if Dataset is not provided.
+                Optional. The fraction of the input data that is to be used to train
```

Comment:
There should be additional documentation at the top of all public `run` method docstrings describing the different splits. Right now, It' just  the fraction split that has that treatment.